internal/schema/schema.go

```diff
@@ -511,10 +512,24 @@ func parseDirectiveDef(l *common.Lexer) *types.DirectiveDefinition {
 		l.ConsumeToken(')')
 	}
 
-	l.ConsumeKeyword("on")
+	switch x := l.ConsumeIdent(); x {
+	case "on":
+		// no-op; Go doesn't fallthrough by default
```

Comment:
Shouldn't you `falltthrough` here? What if a directive has only `on`, without `repeatable`?

Reply:
`on` got consumed by the `ConsumeIdent()` on the previous line, didn't it? Looking at the implementations of `ConsumeIdent` / `ConsumeKeyword`, it looks like they both call `TokenText()` and then `ConsumeWhitespace()`, so I expect they will consume the same number of tokens; and if this code didn't consume `on`, then I would expect `on` to be interpreted as the location, which would make [this check](https://github.com/speezepearson/graphql-go/blob/05a893c8fd6e44c183b68e2d2672f35522d3ca01/internal/schema/schema.go#L526) fail in [this test](https://github.com/speezepearson/graphql-go/blob/05a893c8fd6e44c183b68e2d2672f35522d3ca01/internal/schema/schema_test.go#L880).